### ISSUE TITLE
Selectable blueprints when creating a new form

### ISSUE THREAD

**Issue by motion-work** [statamic/ideas#838]
When creating a new form `/cp/forms/create`, it would be great to have predefined blueprints to select from. I'm thinking of a dropdown like in the form config page where we can select the mail template.

For example we could create a form XY, with an email field predefined and another with the phone field.

Is something like this already in the planning?

**Comment by jasonvarga** [comment]
Not something we've been planning. We don't provide predefined blueprints anywhere else. You have complete control over them.

You could just copy/paste a yaml file from another project.

**Comment by motion-work** [comment]
Not sure what you mean by "We don't provide predefined blueprints anywhere else". When I want to create a new page, I already have the ability to select a predefined blueprint. Like so:
<img width="292" alt="Screenshot 2022-08-02 at 16 56 05" src="https://user-images.githubusercontent.com/11538632/182405686-92da3e7d-d88c-414b-9d29-ca49707f7e8b.png">

It would be great to have the exact same functionality when creating forms. Also a normal user (our clients), who is not an developer can't copy/paste a yaml file.

**Comment by jasonvarga** [comment]
Oh, you're letting your client create new forms?

**Comment by motion-work** [comment]
Yes, in our case we want the client to be able to create his own forms. Here's an example: 

They want to create a public event page, where users can sign up for an event XY. This sign up form can vary from event to event (always different fields are required). For some special events, he needs to collect the mail address from the user. But it's not trivial just to say "Hey, don't forget to add the email field in the form builder". We want it be predefined for him. So he can select a "Event Form Blueprint" or "Team Event Form Blueprint".

**Comment by morhi** [comment]
I have a similar problem. Our customers need to create their own forms, so I have created a collection that is used as a form builder. These entries are then used to create an actual Statamic form. I can't offer my clients to use the built-in form builder because it either provides too many options, too few options, or because the idea of Blueprint (fields) doesn't necessarily apply to front-end work. For example, I need a form field to store more attributes because of design requirements, like pre-defined widths or other design options. One nice feature I added is that the user can define a custom HTML mail and use all the form fields as variables. This is all a bit hacky, but it works.

@motion-work We could have a private discussion on this or other Statamic related stuff, if you like. You are also from the same beautiful city in Switzerland, as I see, so you could come by for a coffee ;)

**Comment by motion-work** [comment]
Hi Maurice. That's a really creative use of collections! I'll definitely hit you up via mail! Hope to see you soon here in Berne.
